Updater.js

```diff
@@ -59,7 +67,7 @@ define(['./util/lang', './Variable'], function (lang, Variable) {
 		},
 		updated: function (updateEvent, by, context) {
 			if (!this.invalidated) {
-				if (!context || context == this.element || context.contains(this.element)) {
+				if (!context || this.contextMatches(context)) {
```

Comment:
could probably be improved by returning the matched subset to limit extraneous renders